controllers/userController.js

```diff
@@ -0,0 +1,19 @@
+const userService = require('../services/userService')
+const { catchAsync } = require('../utils/error')
+
+
+const signUp = catchAsync(async (req, res) => {
+  const { userName, password, email, phoneNumber, address, birth, gender, point} = req.body;
+  
+  if ( !userName || !password || !email || !phoneNumber || !address || !birth || !gender || point === undefined){
+    return res.status(400).json({ message: "KEY_ERROR" });
+  }
+  const insertId = await userService.signUp(userName, password, email, phoneNumber, address, birth, gender, point);
+  res.status(201).json({ insertId });
```

Comment:
insertId는 DB에 생성된 user에 고유 id인데, 이것을 굳이 프론트엔드에게 전달할 필요는 없을 것 같습니다.
다음과 같이 메시지를 수정해 주세요.
```suggestion
  res.status(201).json({ message: 'Success' });
```